<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>454</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>2</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius: 10px;
	border: 2px solid rgb(40,40,40);
	color: rbg(200,200,200);
}
QLineEdit:hover{
	background-color: rgb(66, 239, 255);
	border-radius: 10px;
	border: 2px solid rgb(40,40,40);
}
QLineEdit:focus{
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
	color: rbg(200,200,200);
}</string>
    </property>
    <property name="placeholderText">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius: 10px;
	border: 2px solid rgb(40,40,40)
}
QLineEdit:hover{
	background-color: rgb(66, 239, 255);
	border-radius: 10px;
	border: 2px solid rgb(40,40,40);
}
QLineEdit:focus{
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
	border: 2px solid rgb(200,200,200)
}
QPushButton:hover{
	background-color: rgb(66, 239, 255);
	border-radius: 10px;
	border: 2px solid rgb(40,40,40);
}
QPushButton:pressed{
	border-radius: 10px;
	border: 2px solid rgb(0, 111, 166);
	color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_retorune">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Invalid User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NÃ¢o possui cadastro?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_signin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
	border: 2px solid rgb(200,200,200)
}
QPushButton:hover{
	background-color: rgb(66, 239, 255);
	border-radius: 10px;
	border: 2px solid rgb(40,40,40);
}
QPushButton:pressed{
	border-radius: 10px;
	border: 2px solid rgb(0, 111, 166);
	color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>SIGN IN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
